modules/optimization/examples/materialFrequency/wave1D/tests

```diff
@@ -0,0 +1,15 @@
+[Tests]
+  design = 'GeneralOptimization.md MultiAppReporterTransfer.md ParsedVectorReporter.md ParsedScalarReporter.md OptimizationData.md'
+  issues = '#28372'
+  [wave1D]
+    type = CSVDiff
+    input = inversion.i
+    csvdiff = inversion_out_OptimizationReporter_0001.csv
+    abs_zero = 1e-5
+    recover = False
+    requirement = 'The system shall be able to perform frequency domain material inversion for a bimaterial problem with multiple sets of experimental data.'
+    method = '!dbg'
```

Comment:
Does this prevent dbg from running? If so why?

Reply:
test too slow is only good reason

Reply:
It runs fast so I removed this.  I also removed the valgrind.